templates/custom_data.sh.tpl

```diff
@@ -240,6 +240,12 @@ api_addr      = "https://$FULL_HOSTNAME:${vault_port_api}"
 cluster_addr  = "https://$FULL_HOSTNAME:${vault_port_cluster}"
 
 plugin_directory = "$VAULT_DIR_PLUGINS"
+
+telemetry {
+  disable_hostname = true
+  prometheus_retention_time = "24h"
+}
+
```

Comment:
```suggestion

telemetry {
  disable_hostname = true
  prometheus_retention_time = "24h"
}

``` please remove and we will raise an additional issue for variable to expose telemetry.